### PR TITLE
Added build options to increase scalability

### PR DESCRIPTION
**Pull request by timwright12**
## Description
In testing the scalability of our build system within CMS Export, we found that some updates were needed. These are those updates.

## Testing done
Running successful build and tests

### CODE REVIEW

File: src/site/stages/build/drupal/metalsmith-drupal.js [hunk 1]
```diff
@@ -59,6 +59,7 @@ function pipeDrupalPagesIntoMetalsmith(contentData, files) {
     const pageCompiled = compilePage(page, contentData);
     const drupalPageDir = path.join('.', drupalUrl);
     const drupalFileName = path.join(drupalPageDir, 'index.html');
+    const layoutName = entityBundle || 'page';
```
Comment (timwright12) [inline src/site/stages/build/drupal/metalsmith-drupal.js:62]: We had a few failures when the entityBundle was undefined, it couldn't pick a template. This just sets "page" as the default.

File: src/site/filters/liquid.js [hunk 2]
```diff
@@ -12,6 +12,10 @@ function getPath(obj) {
 module.exports = function registerFilters() {
   const { cmsFeatureFlags } = global;
 
+  // Set timeout option to something higher (20mins)
+  // eslint-disable-next-line no-new
+  new liquid.Context({ timeout: 1200000 });
```
Comment (timwright12) [inline src/site/filters/liquid.js:17]: As the node count increased, it took the templates too long to parse, so just increasing the timeout from the default  of 2 mins set in Tiny Liquid

Comment (ncksllvn) [reply src/site/filters/liquid.js:17]: I'm familiar with this one 

```
Error: Timeout.
    at Timeout.<anonymous> (/vets-website/node_modules/tinyliquid/lib/index.js:79:16)
    at listOnTimeout (internal/timers.js:554:17)
    at processTimers (internal/timers.js:497:7)
```

it doesn't need to be assigned to anything? `liquid.context = new Context...`?

**Review by ncksllvn — APPROVED** [review]

File: src/site/filters/liquid.js (hunk 2 above)
Comment (timwright12) [reply src/site/filters/liquid.js:17]: I tried that initially and it was throwing errors, maybe just something buried in the plugin

**Review by jhonny218 — APPROVED** [review]
LGTM

**Review by mpelzsherman — APPROVED** [review]

**Pull request merged**